plugins/out_stackdriver/stackdriver.h

```diff
@@ -50,6 +50,8 @@
 #define MONITORED_RESOURCE_KEY "logging.googleapis.com/monitored_resource"
 #define LOCAL_RESOURCE_ID_KEY "logging.googleapis.com/local_resource_id"
 #define DEFAULT_LABELS_KEY "logging.googleapis.com/labels"
+#define DEFAULT_SEVERITY_KEY "logging.googleapis.com/severity"
```

Comment:
Should we keep the `log_name_key` in the same `logging.googleapis.com` namespace (similar to the other Google Cloud Logging LogEntry fields https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry) for consistence and also to help avoid conflicting with user / custom labels?

Reply:
I was thinking originally that if we don't have a `log_name_key` set, then we should use the default way of setting the `logName`, but now when I think again, if we don't have it under `logging.googleapis.com/logName`, it will work the same way. so yes, I can add this one.

Reply:
done